cli/commands/init.ts

```diff
@@ -55,13 +59,33 @@ async function generate(
         await writeFile(outputDir, contentPath, content, options);
       },
     },
+    canvas: {
+      create: createCanvas,
+    },
```

Comment:
I think the canvas should be opt-in when using the CLI. At least for now.

Reply:
Now it is only generated if unstable is enabled,  though it makes the canvas lib not get bundled, not sure how to solve it.